plugins/opsgenie/unit_test/test_get_on_calls.py

```diff
@@ -0,0 +1,106 @@
+        self.connection.connect({Input.API_KEY: "1234567e-123c-123c-123c-1234567e9xAd"})
+
+        self.action = GetOnCalls()
+        self.action.connection = self.connection
+        self.action.logger = logging.getLogger("action logger")
+
+        self.params = {"scheduleIdentifier": STUB_SCHEDULE_ID}
+
+    @mock.patch("requests.request", side_effect=mock_request_202)
+    def test_get_on_calls_when_status_ok(self, mock_get):
+        response = self.action.run(self.params)
+        expected_response = {
+            Output.DATA: {
+                "_parent": {"id": "d875alp4-9b4e-4219-alp3-0c26936d18de", "name": "ScheduleName", "enabled": True},
+                "onCallParticipants": [
+                    {"id": "c569c016-alpc-4e20-8a28-bd5dc33b798e", "name": "TeamName", "type": "team"},
+                    {
+                        "id": "15445alp-e46c-446f-9236-7ad89ad1a4f7",
+                        "name": "TeamName_escalation",
+                        "type": "escalation",
+                        "onCallParticipants": [
+                            {
+                                "id": "e55700e1-ff76-4cd0-a6e8-e1a982423alp",
+                                "name": "TeamName_schedule",
+                                "type": "schedule",
+                                "escalationTime": 0,
+                                "notifyType": "default",
+                            },
+                            {
+                                "id": "e55700e1-ff76-4cd0-a6e8-e1a982423alp",
+                                "name": "TeamName_schedule",
+                                "type": "schedule",
+                                "escalationTime": 5,
+                                "notifyType": "next",
+                            },
+                            {
+                                "id": "c569c016-alpc-4e20-8a28-bd5dc33b798e",
+                                "name": "TeamName",
+                                "type": "team",
+                                "onCallParticipants": [
+                                    {
+                                        "id": "balp7783-a9f1-40e3-940c-ffde45656054",
+                                        "name": "user5@opsgenie.com",
+                                        "type": "user",
+                                    },
+                                    {
+                                        "id": "4falpb2e-348d-4b7c-b71b-149efb8361e4",
+                                        "name": "user4@opsgenie.com",
+                                        "type": "user",
+                                    },
+                                ],
+                                "escalationTime": 10,
+                                "notifyType": "all",
+                            },
+                        ],
+                    },
+                ],
+            },
+            Output.TOOK: 0.305,
```

Comment:
making a note that 'took' here might need to change if you op to use elapsed_time instead